packages/common/src/services/explore/Explore.ts

```diff
@@ -50,42 +45,20 @@ export class Explore {
   }
 
   /** TRACKS ENDPOINTS */
-  async getTopUserListens(): Promise<TopUserListen[]> {
-    try {
-      const { data, signature } = await this.audiusBackendInstance.signData()
-      return fetch(
-        `${this.audiusBackendInstance.identityServiceUrl}/users/listens/top`,
-        {
-          headers: {
-            [AuthHeaders.Message]: data,
-            [AuthHeaders.Signature]: signature
-          }
-        }
-      )
-        .then((res) => res.json())
-        .then((res) => res.listens)
-    } catch (e) {
-      console.error(e)
-      return []
-    }
-  }
-
-  // TODO(C-2719)
-  async getTopFolloweeTracksFromWindow(
-    userId: ID,
-    window: string,
-    limit = 25
-  ): Promise<UserTrack[]> {
+  async getBestNewReleases({
+    currentUserId,
+    window = BestNewReleasesWindowEnum.Month
+  }: {
+    currentUserId: ID
+    window?: BestNewReleasesWindowEnum
+  }): Promise<UserTrackMetadata[]> {
     try {
-      const encodedUserId = encodeHashId(userId)
-      const libs = await this.audiusBackendInstance.getAudiusLibs()
-      const tracks = await libs.discoveryProvider.getBestNewReleases(
-        encodedUserId,
+      const sdk = await this.audiusSdk()
+      const { data = [] } = await sdk.full.tracks.bestNewReleases({
```

Comment:
oof, is that method called `bestNewReleases`. Should probably have a `get` in front of it, since that's our convention for all of the read methods.

Reply:
Also seems like we're dropping the `limit` here? Was that intentional?

Reply:
nope lol